doc/GeoIP.conf.md

```diff
@@ -0,0 +1,62 @@
+# GeoIP.conf
+
+## NAME
+
+GeoIP.conf - Configuration file for geoipupdate
+
+## SYNOPSIS
+
+This file allows you to configure your `geoipupdate` program to
+download GeoIP2, GeoLite2, and GeoIP Legacy databases.
+
+## DESCRIPTION
+
+The file consists of one setting per line. Lines starting with `#`
+are comments and will not be processed. All setting keywords are case
+sensitive.
+
+### Required settings:
+
+* `AccountID` - Your MaxMind account ID. This was formerly known as
+  `UserId`.
+* `LicenseKey` - Your case-sensitive MaxMind license key.
+* `EditionIDs`  -List of database edition IDs. Edition IDs may consist
+  of letters, digits, and dashes (e.g., "GeoIP2-City", "106"). Note: this
+  was formerly called `ProductIds`.
+
+### Optional settings:
+
+* `DatabaseDirectory` - The directory to store the database files. If this
+  is not set, you must specify the directory using the `-d` command line
+  argument.
+* `Host` - The host name of the server to use. The default is
+  `updates.maxmind.com`.
+* `Proxy` - The proxy host name or IP address. You may optionally specify
+  a full proxy URL, such as `http://user:pass@127.0.0.1:88888`, or a port
```

Comment:
I wonder whether we need to document this support. I suppose it doesn't hurt.

I might quibble about 88888 not being a valid port at least :slightly_smiling_face:

Reply:
Woops. Do you think we should drop the URL thing? Should we mention `socks5` at all?

Reply:
My thought here was along the lines of documenting support only for what we did before, and not committing ourselves to these other things in perpetuity, some of which it looks like worked only by chance. Plus it keeps the docs a little simpler. But that's just a thought, I don't feel strongly. Up to you!